README.md

```diff
@@ -6,99 +6,155 @@
   - Deserialization
   - Serialization
 
-  ## Schema
-
-  First step for building the schema of your data is creating the schema definition.
-  This can be achived by using the `Parameter.Schema` macro.
-  ```elixir
-  defmodule UserSchema do
-    use Parameter.Schema
-
-    param do
-      param :first_name, :string, key: "firstName", required: true
-      param :last_name, :string, key: "lastName", required: true, default: ""
-      param :age, :integer
-      param :address, {:map, AddressSchema}, required: true
-    end
+## Schema
+
+First step for building the schema of your data is creating the schema definition.
```

Comment:
typos
```
The first step for building a schema for your data is to create a schema definition.
This can be achieved by using the `Parameter.Schema` macro.
```